src/models/__tests__/Template.spec.ts

```diff
@@ -1,5 +1,21 @@
 import casual from 'casual';
-import { Template, Visibility } from "../Template";
+import { Template, TemplateVisibility } from "../Template";
```

Comment:
Tests for "findById", "findByNameAndOwnerId", and "findByUser" are missing.

Reply:
I went back and forth on that since they're not really doing much aside from defining the SQL statement and passing it off to the generic MySqlModel functions